Sources/OpenAPI/OpenAPI.swift

```diff
@@ -1,5 +1,14 @@
 import Vapor
```

Comment:
Maybe we could use an `OpenAPIExample` executable target for testing?

Reply:
Yup, I was thinking the same. I'll add one next time I work on this 😄